extensions/skins/conversation_v1/player.html

```diff
@@ -2,107 +2,258 @@
+                </span>
+                <span ng-if="arePreviousResponsesShown">
+                  <span class="glyphicon glyphicon-triangle-bottom" style="font-size:10px; margin-bottom: -4px;"></span>
+                </span>
+              </span>
+            </h4>
+          </div>
 
-            <div ng-if="$last" ng-show="waitingForOppiaFeedback">
-              <div class="conversation-skin-loading-dots">
-                <div class="oppia-loading-dot-one"></div>
-                <div class="oppia-loading-dot-two"></div>
-                <div class="oppia-loading-dot-three"></div>
+          <div class="conversation-skin-tutor-card-middle-section" ng-if="arePreviousResponsesShown">
+            <div ng-repeat="responsePair in activeCard.answerFeedbackPairs track by $index">
+              <div ng-if="!$last">
+                <answer-feedback-pair profile-picture="profilePicture"
+                                      answer="responsePair.learnerAnswer"
+                                      short-answer="responsePair.shortLearnerAnswer"
+                                      feedback="responsePair.oppiaFeedback">
+                </answer-feedback-pair>
               </div>
             </div>
+          </div>
 
-            <div ng-if="stateName && !waitingForNewCard && !waitingForOppiaFeedback && interactionIsInline && $last">
-              <div class="conversation-skin-inline-interaction">
-                <table class="conversation-skin-card-row-container" style="margin-bottom: 0;">
-                  <tr class="conversation-skin-card-row">
-                    <td class="conversation-skin-row-avatar" valign="top">
-                      <img class="conversation-skin-row-avatar-img img-circle" ng-src="<[profilePicture]>">
-                    </td>
-                    <td class="conversation-skin-learner-input protractor-test-conversation-input"
-                        angular-html-bind="inputTemplate">
-                    </td>
-                  </tr>
-                </table>
+          <!-- Always show the most recent response pair, if there is one. -->
+          <div class="conversation-skin-tutor-card-bottom-section" ng-if="activeCard.answerFeedbackPairs.length > 0">
+            <answer-feedback-pair profile-picture="profilePicture"
+                                  answer="activeCard.answerFeedbackPairs[activeCard.answerFeedbackPairs.length - 1].learnerAnswer"
+                                  short-answer="activeCard.answerFeedbackPairs[activeCard.answerFeedbackPairs.length - 1].shortLearnerAnswer"
+                                  feedback="activeCard.answerFeedbackPairs[activeCard.answerFeedbackPairs.length - 1].oppiaFeedback">
+            </answer-feedback-pair>
+          </div>
+
+          <div ng-if="!waitingForOppiaFeedback && (activeCard.interaction || waitingForContinueButtonClick) && currentProgressDotIndex === transcript.length - 1">
```

Comment:
Might want to add another comment here to describe that this div is used for both interaction instructions or a pending continue button.